src/control/server/storage/config.go

```diff
@@ -259,6 +267,177 @@ func (sc *ScmConfig) Validate(class Class) error {
 	return nil
 }
 
+type stringSet map[string]struct{}
```

Comment:
Oops, just noticed that this is a dupe of [common.StringSet](https://github.com/daos-stack/daos/blob/master/src/control/common/collection_utils.go#L20), added after I started this PR ages ago. I'll wait to see what kind of feedback I get before pushing an update.